Cargo.toml

```
@@ -138,7 +138,7 @@ instant = { version = "0.1.13", default-features = false }
 tokio-util = "0.7.12"
 
 # Substrate crates:
-sp-core = { version = "31.0.0", default-features = false }
+sp-core = { version = "34.0.0", default-features = false }
 sp-crypto-hashing = { version = "0.1.0", default-features = false }
 sp-runtime = "34.0.0"
 sp-keyring = "34.0.0"
```

```suggestion
sp-runtime = "39.0.0"
sp-keyring = "39.0.0"
```

I think that'll do it